### DISCUSSION TITLE
Does huly support "Copy link" from the messages?

### DISCUSSION THREAD

**Discussion by perperj-super** [hcengineering/platform#5653] [Q&A] [answered]
Hello!

I noticed huly is already very similar to slack and just wanted to know if it supports "Copy link" from the message?

![image](https://github.com/hcengineering/platform/assets/101011021/52c87aaf-abce-4908-bf51-e8689c935bce)

**Comment by BykhovDenis** [comment, marked as answer]
Hello, yes we offer this functionality.
